ibis/expr/operations.py

```diff
@@ -1209,7 +1209,36 @@ class CumulativeMin(CumulativeOp):
     output_type = rlz.array_like('arg')
 
 
+class CumeDist(AnalyticOp):
```

Comment:
can you show me an authoritative source that actually uses this in real code. 

How is having this *in addition* to percent_rank useful at all? I would rather have a real use case and not add this at this time.

Reply:
I think we are mixing two different things in this PR. That maybe makes sense since they are related.

- We are adding a new window function cume_dist.
- We are fixing percent_rank, since Ibis percent_rank is different from the sql one

@xmnlab does it make sense to split this, and go step by step, so we have more focused discussions on the changes? Start by the percent_rank, making the proposed changes, and explaining why you think it needs to be fixed. Once that is merged, let's go back to cume_dist. It's been almost 6 months discussing this, and if we don't split things it's difficult to recall exactly what has been discussed.

Thanks!

Reply:
thanks @datapythonista I will work on that.